src/constants/index.ts

```diff
@@ -64,7 +64,7 @@ export const GENERAL_PLOT_SETTINGS = {
         t: 10,
     },
     moveDropdownCutoffWidth: 370,
-    plotHeight: 580,
+    heightMargin: 56 + 74 + 140, // header height + tab height + margins
```

Comment:
kinda scary looking. do these numbers show up individually anywhere else?

Reply:
yeah, so they're defined as constants in the CSS, which is why I left them as separate numbers here. I could make them separate constants in the javascript, but not not sure if that makes it much better